backend/alembic/versions/e1e7929bebd8_initial_migration.py

```
@@ -0,0 +1,49 @@
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('question')
+    op.drop_index('ix_quiz_category', table_name='quiz')
+    op.drop_table('quiz')
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    """Downgrade schema."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('quiz',
+    sa.Column('id', sa.INTEGER(), autoincrement=True, nullable=False),
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)